Troubleshoot with PBID on the gateway computer (#224)
MicrosoftDocs/data-integration: data-integration/gateway/service-gateway-tshoot.md

![On-premises data gateway event logs.](media/service-gateway-tshoot/on-prem-data-gateway-event-logs.png)

## Troubleshooting refresh failures for a specific source

Refreshes utilizing the gateway require that the source be accessible from the computer with the gateway installation.

> As such, when troubleshooting a data source, especially one that requires additional components to be installed on the computer (such as a third party database driver) or additional configuration (such as access to a network drive file or folder), it can be helpful to troubleshoot with Power BI Desktop from the gateway computer.

Reviewer: The sentence is too long for translation. Simpler is better - split it into two.

```suggestion
To troubleshoot a data source issue, use Power BI Desktop locally on the gateway computer to test the connection. This test is especially helpful if the data source requires additional components to be installed on the computer, such as a third-party database driver. Also, a local test helps narrow down if the data source connection requires additional environment settings or access to a shared network drive file or folder.
```

> Using Power BI Desktop on the gateway computer allows for more rapid iterations of testing the connection and configuring any requirements on the gateway computer directly.

Reviewer: ```suggestion
This technique allows you to test iteratively, testing the connection on the gateway computer after each configuration change.
```

![On-premises data gateway event logs.](media/service-gateway-tshoot/on-prem-data-gateway-event-logs.png)

## Troubleshoot refresh failures for a specific source

Refreshes utilizing the gateway require that the source be accessible from the computer with the gateway installation.
To troubleshoot a data source issue, use Power BI Desktop locally on the gateway computer to test the connection.
This test is especially helpful if the data source requires additional components to be installed on the computer, such as a third-party database driver. Also, a local test helps to test data source connections that require additional environment settings, such as access to a shared network drive file or folder.
This technique allows you to test iteratively, testing the connection on the gateway computer after each configuration change.

> Although it is not a guarantee of a successful refresh through the gateway, a successful Power BI Desktop refresh from the gateway computer is a strong indicator that everything is configured correctly on the gateway computer.

Reviewer 2: Microsoft style requires the use of contractions (for more friendly text), so change "is not" to "isn't".

Author: "is not" -> "isn't" in latest commit

> In other words, if you cannot refresh in Power BI Desktop from the gateway computer, it is highly unlikely that a refresh through the gateway will succeed.

Reviewer 2: Again, use contractions - change "cannot" to "can't" and "it is" to "it's".
